nft_ingester/src/processors/account_based/mpl_core_processor.rs

```diff
@@ -133,7 +134,14 @@ impl MplCoreProcessor {
             })
             .unwrap_or((0, &default_creators));
 
-        let mut plugins_json = serde_json::to_value(&asset.plugins)
+        // convert HashMap plugins into BTreeMap to have always same plugins order
+        // for example without ordering 2 assets with same plugins can have different order saved in DB
+        // it affects only API response and tests
+        let ordered_plugins: BTreeMap<_, _> = asset.plugins
+            .iter()
+            .map(|(key, value)| (format!("{:?}", key), value))
```

Comment:
why format here? doesn't serve properly encode the PluginType? Looks like `asset.plugins.iter().collect()` should have worked, didn't it?

Reply:
no it's not. Because HashMap as a key has `PluginType` type and that type doesn't implement `Ord`. But this convertion requires this type to be `Ord`